pkg/mount/mountinfo_linux.go

```diff
@@ -26,6 +30,8 @@ const (
 	   (10) mount source:  filesystem specific information or "none"
 	   (11) super options:  per super block options*/
 	mountinfoFormat = "%d %d %d:%d %s %s %s %s"
+
+	TMPFS_MAGIC = 0x1021994
```

Comment:
Should probably reuse `github.com/containers/storage/drivers.FsMagicTmpfs` and the adjacent `GetFSMagic()` function here.

Reply:
should I move these functions to `github.com/containers/storage/drivers`?  Otherwise it introduces an import cycle.

Reply:
Maybe pkg/util?

Reply:
Never mind that does not exists.

Reply:
Seems to make sense.